wallets/metamask/src/fixtures/metaMaskFixtures.ts

```diff
@@ -11,13 +11,22 @@ import {
 } from '@synthetixio/synpress-cache'
 import { type Anvil, type CreateAnvilOptions, createPool } from '@viem/anvil'
 import fs from 'fs-extra'
+import { loadEnv } from '../config'
+import { importAndConnectForFixtures } from '../fixture-actions/importAndConnectForFixtures'
+import { cachelessSetupMetaMask } from '../fixture-actions/noCacheMetaMaskSetup'
 import { persistLocalStorage } from '../fixture-actions/persistLocalStorage'
+import { SEED_PHRASE } from '../utils/constants'
+
+loadEnv()
+
+const USECACHE: boolean = process.env.SYNPRESS_USE_CACHE === undefined || process.env.SYNPRESS_USE_CACHE === 'true'
```

Comment:
it will be easier to use if we export all the variables based on the env variable from a one file